app/src/main/res/layout/activity_book.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<android.support.design.widget.CoordinatorLayout
```

Comment:
tutaj troszkę zmian bo nacisnęłąm auto formatowanie pliku.

Reply:
najważniejsze to w sumie ustawiłam nasz placeholder na src dla imageview, dzięki czemu nie trzeba go ładować z glide :)